doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -307,6 +308,31 @@ response from the NCP acknowledging the command (with `CMD_VALUE_IS`).
 Once that acknowledgement is received the host may enter the low-power
 state.
 
+### PROP 4104: PROP_UNSOL_UPDATE_FILTER {#prop-unsol-update-filter}
+
+* Required only if `CAP_UNSOL_UPDATE_FILTER` is set.
+* Type: Read-Write
+* Packed-Encoding: `A(I)`
+* Default value: Empty.
+
+Contains a list of properties which are excluded from generating
+unsolicited value updates. This property **MUST** be empty after reset.
+
+Hosts **SHOULD NOT** add properties to this list which are not
+present in `PROP_UNSOL_UPDATE_LIST`. If such properties are added,
+the NCP **MUST** ignore the unsupported properties.
+
+### PROP 4105: PROP_UNSOL_UPDATE_LIST {#prop-unsol-update-list}
+
+* Required only if `CAP_UNSOL_UPDATE_FILTER` is set.
+* Type: Read-Only
+* Packed-Encoding: `A(I)`
+
+Contains a list of properties which are capable of generating
+unsolicited value updates. This list can be used when populating
+`PROP_UNSOL_UPDATE_FILTER` to disable all unsolicited property
+updates.
+
```

Comment:
> Am I interpreting this correctly? `PROP_UNSOL_UPDATE_LIST` is a constant for a given NCP image. Any changes to `PROP_UNSOL_UPDATE_FILTER` will not affect the value of `PROP_UNSOL_UPDATE_LIST`.

Correct. `PROP_UNSOL_UPDATE_LIST` is effectively a constant.

> This therefore allows one to read `PROP_UNSOL_UPDATE_LIST`, remove the properties from the result that one desires to continue receiving, and writing the result back to `PROP_UNSOL_UPDATE_FILTER`?

Yep.

> Adhering to this behavior should prevent one from supplying a property that is not supported by this feature as well as providing a potentially useful reference as to all the possible unsolicited properties one can expect to receive.

Yep.